src/js/shared/status-constants.js

```diff
@@ -24,6 +25,7 @@ module.exports = {
     INCOMPLETE: 'incomplete',
     COMPLETE_YES: 'complete-yes',
     COMPLETE_NO: 'complete-no',
+    COMPLETE_EXPIRED: 'complete-expired',
```

Comment:
Surely their rsvp is complete, though? This seems as valid as a completion as a "no" is.